### ISSUE TITLE
Free-form descriptions under Feature, Scenario, Background, etc. should be wrapped using the printWidth prettier option

### ISSUE THREAD

**Issue by lcrespom** [mapado/prettier-plugin-gherkin#22]
The text in free-form descriptions should take the `printWidth` prettier option into account. For example:

```
Feature: lorem ipsum

Dolor sit amet consectetur qui dolorem ipsum, quia dolor sit amet consectetur adipisci velit, sed quia non numquam eius modi tempora incidunt, ut labore et dolore magnam aliquam quaerat voluptatem (etc...)
```
If this option is set in `.prettierrc`: `"printWidth": 80`

Then the text should be wrapped accordingly:

```
Feature: lorem ipsum

    Dolor sit amet consectetur qui dolorem ipsum, quia dolor sit amet
    consectetur adipisci velit, sed quia non numquam eius modi tempora
    incidunt, ut labore et dolore magnam aliquam quaerat voluptatem (etc...)

```

P.S. thank you for this very useful plugin :-)

**Comment by jdeniau** [comment]
That you and nice catch!

**Comment by jdeniau** [comment]
@lcrespom released as 2.4.0

**Comment by lcrespom** [comment]
Thank you very much. I checked it and it works.

Regards,
Luis.
